getting-started/macos/macos-sage.md

```diff
@@ -0,0 +1,38 @@
+---
+ID:
+post_title: Sage development environment for macOS
+author: Michael W. Delaney
+post_excerpt: ""
+layout: doc
+permalink: >
+
+published: false
+post_date:
+---
+# Requirements
+
+---
+
+Sage relies on a few build tools to manage dependencies and build assets. Install these tools:
+
+- nvm
+- Node.js < 10.0
+- yarn
+
+## nvm
+
+Install nvm from the instructions at [https://github.com/creationix/nvm](https://github.com/creationix/nvm), or with the following command:
+
+    $ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+
+## Node.js
+
+Install node.js < 10.0 from nvm:
+
+    $ nvm install --lts
```

Comment:
Not an issue now, but just noting that in Oct 2018 10.x will move to LTS, so depending on whether Sage has Node 10.x compatibility, we may need to change this to a specific version rather than using the `--lts` tag.

Reply:
Perhaps recommending `nvm install lts/carbon` would be better.

Reply:
If that's the correct command to get the version we need I'm all for it.